app/Http/Controllers/ProfileController.php

```diff
@@ -216,4 +219,49 @@ public function getMenuState(Request $request)
             $request->session()->put('menu_state', 'closed');
         }
     }
+
+
+    /**
+     * Print inventory
+     *
+     * @author A. Gianotto
+     * @since [v6.0.12]
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function printInventory()
+    {
+        $show_user = User::where('id', Auth::user()->id)->withTrashed()->first();
+        $assets = Asset::where('assigned_to',  Auth::user()->id)->where('assigned_type', User::class)->with('model', 'model.category')->get();
+        $accessories = $show_user->accessories()->get();
+        $consumables = $show_user->consumables()->get();
+
+        return view('users/print')->with('assets', $assets)
+            ->with('licenses', $show_user->licenses()->get())
```

Comment:
If you want to prepopulate variables above, you should add this one up there too (along with `$accessories` and `$consumables`), or you should do the same with them, down here. Either way. But consistency is helpful.